gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Jul 12 20:56:16 CEST 2017
+#Sat Oct 07 13:45:45 PDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=http\://services.gradle.org/distributions/gradle-4.0.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.0.1-bin.zip
```

Comment:
:+1:

Reply:
`https` doesn't work with the JDK 7 build.

Reply:
Hm this change must have happened automatically somehow, perhaps intellij did it?
I'll remove any gradle-wrapper touching things from this PR

Reply:
@ktoso I believe this line is breaking Travis. Revert?

Reply:
@ktoso as in, revert the Gradle-script-related changes in this PR